theano/tensor/tests/test_opt.py

```diff
@@ -5584,6 +5654,37 @@ def test_local_sum_sum_dtype(self):
         finally:
             config.on_opt_error = backup
 
+    def test_stack_trace(self):
+        """
+        Test that stack trace is copied over correctly.
+        """
```

Comment:
Actually, we now prefer to have a comment rather than a docstring, since nosetests annoyingly replaces the test name by the docstring when displaying which test is being executed.